src/app/loadout/loadout.service.ts

```diff
@@ -21,7 +21,19 @@ export enum LoadoutClass {
   hunter = 2
 }
 
-type LoadoutItem = DimItem;
+export const getLoadoutClassDisplay = (loadoutClass: number) => {
```

Comment:
Same comment on functions vs const arrow functions, and `LoadoutClass`.

Reply:
Done.